docs/INSTALL-CMAKE.md

```diff
@@ -294,6 +294,7 @@ Details via CMake
 - `USE_NGHTTP2`:                            Use nghttp2 library. Default: `ON`
 - `USE_NGTCP2`:                             Use ngtcp2 and nghttp3 libraries for HTTP/3 support. Default: `OFF`
 - `USE_QUICHE`:                             Use quiche library for HTTP/3 support. Default: `OFF`
+- `USE_SSLS_EXPORT`:                        Enable experimental SSL session import/export
```

Comment:
```suggestion
- `USE_SSLS_EXPORT`:                        Enable experimental SSL session import/export. Default: `OFF`
```
Also I suggest moving to the end of the `## Enabling features` section.

Reply:
Done.